CHANGES.rst

```diff
@@ -49,6 +49,7 @@ Improvements
 - Add background color option and layer order to badge/poster designer items (:pr:`5139`,
   thanks :user:`SegiNyn`)
 - Allow external users in event/category ACLs (:pr:`5146`)
+- Add support for abstract cloning (:pr:`5217`)
```

Comment:
This is for 3.1.1, not 3.1 (already released). Once you rebase, the changelog should include the 3.1.1 section